web/server.go

```diff
@@ -172,7 +173,8 @@ func NewMux(s *Server) (*http.ServeMux, error) {
 		mux.HandleFunc("/print", s.servePrint)
 	}
 	if s.RPC {
-		mux.Handle(rpc.DefaultRPCPath, rpc.Server(traceAwareSearcher{s.Searcher}))       // /rpc
+		mux.Handle(rpc.DefaultRPCPath, rpc.Server(traceAwareSearcher{s.Searcher})) // /rpc
+		mux.Handle("/api/", http.StripPrefix("/api", zjson.JSONServer(traceAwareSearcher{s.Searcher})))
```

Comment:
nit add a variable to `json` for `DefaultAPIPath` to match the style of the other endpoints.

Reply:
Well, this line works only when passing exactly `/api/` to Handle and `/api` to StripPrefix.  I originally did what you said, but I thought string literals were much clearer in the end.